examples/subplots_axes_and_figures/basic_gridspecs.py

```diff
@@ -0,0 +1,107 @@
+"""
+===================================================
+Advanced subplot placement with the gridspec module
+===================================================
+
+The ``subplots`` function is good for building a uniform grid
```

Comment:
Should use single backquotes here and below and link to the actual function/module.  The additional sentence regarding the API doc would then become unneeded.